### DISCUSSION TITLE
前端配置的“昵称（NickName）”始终显示“匿名”

### DISCUSSION THREAD

**Discussion by eddiehe99** [walinejs/waline#953] [Q&A] [answered]
在前端配置中无论设置“lang”为什么语言，昵称输入框中始终默认填写了中文“匿名”，在文档中好像也找不到修改指引。

希望设置了“lang”为不同语言后，“匿名”也能跟随变换为相应的语言。

**Comment by Mister-Hope** [comment, marked as answer]
这是浏览器自动补全的问题
